src/editor-sidebar/create-panel.js

```diff
@@ -0,0 +1,260 @@
+import { __ } from '@wordpress/i18n';
+import { useState } from '@wordpress/element';
+import { useDispatch, useSelect } from '@wordpress/data';
+import apiFetch from '@wordpress/api-fetch';
+import { downloadFile } from '../utils';
+import { store as noticesStore } from '@wordpress/notices';
+import {
+	// eslint-disable-next-line
+	__experimentalVStack as VStack,
+	// eslint-disable-next-line
+	__experimentalSpacer as Spacer,
+	// eslint-disable-next-line
+	__experimentalText as Text,
+	// eslint-disable-next-line
+	__experimentalHeading as Heading,
+	// eslint-disable-next-line
+	__experimentalNavigatorToParentButton as NavigatorToParentButton,
+	PanelBody,
+	Button,
+	TextControl,
+	TextareaControl,
+} from '@wordpress/components';
+import { chevronLeft } from '@wordpress/icons';
+
+export const CreateThemePanel = () => {
+	const { createErrorNotice, createInfoNotice } = useDispatch( noticesStore );
+
+	const [ theme, setTheme ] = useState( {
+		name: '',
+		description: '',
+		uri: '',
+		author: '',
+		author_uri: '',
+		tags_custom: '',
+	} );
+
+	useSelect( ( select ) => {
+		const themeData = select( 'core' ).getCurrentTheme();
+		setTheme( {
+			name: themeData.name.raw,
+			description: themeData.description.raw,
+			author: themeData.author.raw,
+			author_uri: themeData.author_uri.raw,
+			theme_uri: themeData.theme_uri.raw,
+			subfolder:
+				themeData.stylesheet.lastIndexOf( '/' ) > 1
+					? themeData.stylesheet.substring(
+							0,
+							themeData.stylesheet.lastIndexOf( '/' )
+					  )
+					: '',
+		} );
+	}, [] );
+
+	const handleExportClick = () => {
+		const fetchOptions = {
+			path: '/create-block-theme/v1/export-clone',
+			method: 'POST',
+			data: theme,
+			headers: {
+				'Content-Type': 'application/json',
+			},
+			parse: false,
+		};
+
+		async function exportCloneTheme() {
+			try {
+				const response = await apiFetch( fetchOptions );
+				downloadFile( response );
+			} catch ( error ) {
+				const errorMessage =
+					error.message && error.code !== 'unknown_error'
+						? error.message
+						: __(
+								'An error occurred while attempting to export the theme.'
+						  );
+				createErrorNotice( errorMessage, { type: 'snackbar' } );
+			}
+		}
+
+		exportCloneTheme();
+	};
+
+	const handleCreateBlankClick = () => {
+		apiFetch( {
+			path: '/create-block-theme/v1/create-blank',
+			method: 'POST',
+			data: theme,
+			headers: {
+				'Content-Type': 'application/json',
+			},
+		} )
+			.then( () => {
+				createInfoNotice(
+					__(
+						'Theme created successfully. The editor will now reload.',
```

Comment:
I would actually prefer to make this notification a blocking one so that there's no option but to dismiss it.  I looked for a native Gutenberg way to do that but I didn't find anything.  Alternately I was also thinking about a good old-fashioned alert.

Reply:
A blocking notification would be better here. Is there maybe a native blocking modal that could be used?

Reply:
I tried to refactor this to use 'alert()`, however the linter complained about using that.  I wasn't able to find anything similar native to gutenberg UI tooling though.

Reply:
I just switched them to `alert()` calls and told eslint to ignore it.  Think that's OK?

Reply:
Just tested this out. I think that works well! We can always revisit it if we create/find a suitable Gutenberg UI. Thanks for doing that!